---
title: "Approaches and methods"
---

```{r, echo = FALSE}
knitr::include_graphics("images/full_length_rna.png")
```
<br>

## Genomics

### Long-read sequencing

In my postdoctoral research, I'm leveraging long-read genomic and transcriptomic data to understand the causes and consequences of gene family expansion and contraction. Exploring a classical example of environmental adaptation, the evolution of antifreeze protein (AFP) genes in polar fishes, I'm using PacBio HiFi data to study structural genomic variation such as copy number variants, translocation, and inversion. 
<br>

In shallow-water, cold-adapted lineages that experienced expansions in AFP copy number, I'm employing full-length long-read RNA-seq to confirm the expression of duplicated AFP genes and understand how gene family expansion has influenced their regulation. 
<br>

In recent work led by undergraduate Owen Moosman, we have developed and applied tools to more accurately study structural variants in alignments of long reads to haplotype-aware data structures such as phased genomes and pangenomes.
<br>

### Epigenomics
The role of epigenomic processes such as DNA methylation in driving changes in gene expression and phenotype is poorly understood. Part of this knowledge gap is attributed to (i) poor understanding of how different types of epigenomic modifications to DNA and chromatin interact and (ii) poor integration of epigenomic and transcriptomic data. I have integrated ATAC-seq with bisulfite sequencing and RNA-seq sampled from purple sea urchins exposed to experimental upwelling to investigate how chromatin accessibility influences associations between differential DNA methylation and expression in response to environmental stress. 
<br>

To improve multiomic studies of DNA methylation and gene expression, I am currently codinge a structural equation modeling approach to test for changes in gene expression that are associated with environmentally-induced changes in DNA methylation. This project leverages whole genome bisulfite sequencing data and RNA-seq from urchin larvae spawned in a quantitative genetic breeding design.  

### Genome assembly
I am leading and aiding in the assembly of chromosome-scale, haplotype-phased reference genomes for a diversity of threatened and elusive marine animals including deep sea fishes and marine mammals. This work combines long-read and HiC reads, as well as the development of new software to evaluate accurate phasing and assembly. 
<br>

## Ecological physiology

### Experimental design

Laboratory experiments are a hallmark of my scientific approach. I frequently employ common garden experiments, multigenerational studies, and quantitative genetic breeding designs under environmentally-manipulated seawater systems. These experiments allow me understand how global change stressors reshape phenotypes, regulation of the genome, and evolutionary processes. 
<br>

### Physiological performance
My research on orgnismal physiology has frequently included phenotyping of fitness-correlating traits (survival and reproduction) and performance traits in ectotherms (upper thermal tolerance, growth rate, biomineralization, development, and metamorphosis). Fo 
<br>

### Integrating physiological, environmental, and genomic data
Integrating environmental and biological data requires statistical analyses go beyond multiple regression, evaluating specific and mechanistic hypotheses constructed from directional relationships among variables. My work in organismal physiology and multiomics employs structural equation modeling to do just this. Using causal inference with structural equations, I evaluate how environmental variation spurs changes in epigenomic variables, gene expression, and phenotypes while controlling for confounding factors that otherwise cause spurious correlation. The goal of this approach is to identify networks and modules of gene regulatory processes that predict plastic responses to the environment.  




